server/src/internalClusterTest/java/org/opensearch/search/scroll/SearchScrollIT.java

```diff
@@ -135,6 +135,7 @@ public void testSimpleScrollQueryThenFetch() throws Exception {
         }
 
         client().admin().indices().prepareRefresh().get();
+        indexRandomForConcurrentSearch("test");
```

Comment:
A few of these tests, like this one, look like the index many documents already. For those that are already indexing a lot of documents do we still need to add this?

Reply:
it doesn't go down the 2 slice path if we dont add indexRandomForConcurrentSearch call.